partials/sections/blog-sidebar/blog-widget.hbs

```diff
@@ -13,47 +13,31 @@
         </form>
     </div>
 </div>
-
+{{#posts}}
 <div class="card mt-5">
     <div class="card-header bg-transparent">
         <span class="h5 m-0">Recent Posts</span>
     </div>
     <div class="list-group list-group-flush">
-        <a href="#" class="list-group-item list-group-item-action d-flex py-3">
```

Comment:
This change needs to go on the detail page not the blog index page.  The recent posts list is the same content as the left-hand side on this page.